docs/DeveloperGuide.md

```diff
@@ -238,7 +238,75 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Review Feature
 
+#### Implementation
+
+The review feature builds upon the `Person` class by adding more functionality to the class. It utilises the `rv/` tag in the `add` command to add a review stored internally as a `Review` object.
+It implements the following operations:
+* `isValidReview()`  — Determines if the input value is valid.
+* `isEmptyReview()`  — Determines if the stored Review is empty (stored as `-No Review-` String).
+
+In addition, numerous classes had to be updated to accommodate the Review feature:
+* `AddCommandParser` to parse the new `rv/` tag.
+* `ParserUtil` to parse the Review content.
+* `EditCommand` and `EditCommandParser` to enable editing of Reviews.
+
+The following sequence diagram gives the overview of the add command with a review:
+
+![Sequence Diagram for Review](images/ReviewSequenceDiagram.png)
+
+The following activity diagram shows how the review operation works:
+
+![Sequence Diagram for Review](images/ReviewActivityDiagram.png)
+
+
+Given below is an example usage scenario of adding a Review:
+
+Step 1. The user adds a new Review using the `add` command e.g. `add n/test e/test@gmail.com p/12344321 a/test c/att rv/test review`
```

Comment:
Maybe you could give some details on how the `add` command is parsed by the `Parser`?

Reply:
hmmm I felt that this was convered in the sequence diagram